http/jsonapi/generator/generate_handler.go

```diff
@@ -565,6 +588,38 @@ func (g *Generator) buildHandler(method string, op *openapi3.Operation, pattern
 	return route, nil
 }
 
+func generateAuthentication(op *openapi3.Operation, secSchemes map[string]*openapi3.SecuritySchemeRef) (*jen.Group, error) {
+	r := &jen.Group{}
+	r.Add(jen.Comment("Authentication Handling "))
+	for _, sl := range *op.Security {
+		for name, val := range sl {
+			securityScheme := secSchemes[name]
+			switch securityScheme.Value.Type {
```

Comment:
I think it would be best to add a default case to the switch in case we encounter an unsupported type. Like maybe panic then.